docs/provenance/exploring-provenance/exploring-record-lineage.md

```diff
@@ -245,6 +245,28 @@ To use the "View Selected Entity" button, first select the node to show it's [de
 | :------------------------------------------------------------------------------------------------------: |
 | <img src="../../assets/images/provenance/exploring/graph/view_entity.png" alt="drawing" width="800"/> |
 
+
+### Advanced Provenance Queries
+
+In addition to interrogation of provenance via normal queries facilitated by the manual expansion of entities, the Provenance System supplies special querying capabilities for automatically discerning affected downstream or upstream Datasets and Agents. These special queries can explore at a greater depth, and filter by relationships and types. There are four types of special queries:
```

Comment:
I think this wording could be simplified. Particularly this bit "interrogation of provenance via normal queries facilitated by the manual expansion of entities, the Provenance System supplies special querying capabilities for automatically discerning affected downstream or upstream Datasets and Agents"

Also maybe we should be consistent re: "advanced" vs "special" queries - I don't mind which.